explainer.md

```diff
@@ -144,7 +144,10 @@ document.getElementById('changeConfigBtn').onclick() = function() {
 
 ```
 partial interface PresentationRequest {
-  constructor(sequence<PresentationSource> sources);
+  // This constructor replaces
+  //   constructor(sequence<USVString> urls);
+  // in the current Presentation API spec.
+  constructor(sequence<(USVString or PresentationSource)> sources);
```

Comment:
Do IDL sequences allow polymorphism, e.g. a mix of USVString and PresentationSource elements?

Would that situation cause any issues with how the request is interpreted by the browser?  For example, what if the same URL appears twice (once by itself, and once in a PresentationSource)?

Reply:
> Do IDL sequences allow polymorphism, e.g. a mix of USVString and PresentationSource elements?

Do you mean a single sequence instance containing both types of elements? Then yes.

> Would that situation cause any issues with how the request is interpreted by the browser? For example, what if the same URL appears twice (once by itself, and once in a PresentationSource)

Off the top of my head I can't think of any issues we're introducing that we don't already have with a sequence of strings. For instance we can already pass in a sequence containing multiple identical or near-identical URL strings into the PresentationRequest ctor.

Reply:
That's true, and the spec says that the browser must use the first source that is compatible with the user's chosen device.